### ISSUE TITLE
Feature request: Widget

### ISSUE THREAD

**Issue by kristofors** [Dimillian/IceCubesApp#859]
It would be handy to have a couple widget options for displaying a selected source of toots: a specific list, notifications, local timeline, direct conversations, etc.

**Comment by JLO64** [comment]
Given the upcoming features for widgets in macOS Sonoma, this feature would be great to have on a desktop as well as on mobile.

**Comment by Shanesan** [comment]
I would love to be able to see the latest images from my home feed on a widget.

**Comment by lumaa-dev** [comment]
Maybe reconsider this idea? Because the official [Mastodon](https://apps.apple.com/fr/app/mastodon-for-iphone-and-ipad/id1571998974) app has home screen widgets

**Comment by dtantono** [comment]
Yeah, But, mastodon client does not offer widget to get latest feeds.
